src/main/resources/ca/corefacility/bioinformatics/irida/database/changesets/19.04/analysis-templates.xml

```diff
@@ -24,26 +24,39 @@
             <column name="automated" type="bit(1)"/>
         </addColumn>
 
-        <!--updating old automated runs-->
-        <sql>UPDATE analysis_submission SET automated=0</sql>
+        <!--Drop old notnulls since the table now has the template also encoded -->
```

Comment:
Could you move this entire file to the database changesets for the latest release?

Reply:
Merged development up and moved to 19.09 in da81690